Reference/Searching/Examine/transforming-index-values.md

```diff
@@ -0,0 +1,120 @@
+---
+versionFrom: 8.0.0
+---
+
+# TransformingIndexValues
+
+In the [Quick Start](Quick-Start/index.md) documentation you can see how to perform a simple search with Examine. That is great if you want to search between node names or you know that you always want to search for a specific field - e.g. `bodyText`. 
```

Comment:
This whole section on TransformingIndexValues seems to indicate that this event is for one single purpose only which it isn't. This is just one example of what you can do with this event. 

Before going into this example we need to explain what this event is actually for.

> The TransformingIndexValues event allows you to manipulate the data that will be indexed during an index operation. With this event you can add, remove or modify the data that is going into the index which can be helpful in many scenarios. 

Then perhaps another sub headline: `Example`